gc/include/omrmm.hdf

```diff
@@ -227,6 +227,8 @@ typedef uintptr_t (*condYieldFromGCFunctionPtr) (OMR_VMThread *omrVMThread, uint
 		<data type="uintptr_t" name="regionSize" description="heap region size" />
 		<data type="uintptr_t" name="regionCount" description="total heap region count" />
 		<data type="uintptr_t" name="arrayletLeafSize" description="arraylet leaf size" />
+		<data type="uint8_t" name="isArrayletDoubleMappingEnabled" description="set to true if arraylet double mapping status is enabled, false otherwise" />
+		<data type="uint8_t" name="isArrayletDoubleMappingRequested" description="set to true if arraylet double mapping was requested by user, false otherwise" />
```

Comment:
You don't have to pass isArrayletDoubleMappingRequested through this hook, since it's in GCExtensions, which is visible in Verbose GC handler.